### PR TITLE
Add Wagtail 2.3 migration to Image model

### PR DESCRIPTION
**Pull request by cwdavies**
Wagtail 2.X adds a new field to the base Wagtail Image model. We need to create a migration for our CFGOVImage model to reflect that change.

## Additions

- file_hash migration to CFGOVImage model

## Testing

1. All tests of CFGOVImage pass
2. All expected behavior of CFGOVImage is retained
3. makemigrations --dry-run --check shows no missing migrations

## Checklist

- [X] PR has an informative and human-readable title
- [X] Changes are limited to a single goal (no scope creep)
- [X] Code can be automatically merged (no conflicts)
- [X] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [X] Passes all existing automated tests
- [X] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [X] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by chosak** [comment]
Why is this change needed? The base `AbstractImage` that our custom image class inherits from gets [`file_hash`](https://github.com/wagtail/wagtail/blob/v2.3/wagtail/images/models.py#L84) in 2.2, so `CFGOVImage` gets it there as well. With this change we'd be redefining this field.

File: cfgov/v1/models/images.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 class CFGOVImage(AbstractImage):
     alt = models.CharField(max_length=100, blank=True)
+
```
Comment (higs4281) [inline cfgov/v1/models/images.py]: do we need an extra space in this spot?

**Comment by willbarton** [comment]
@chosak we'll remove it when we pin Wagtail 2.3 in cfgov-refresh. 

This gets us the migration for CFGOVImage that we will require anyway and lets us test against Wagtail 2.3 right now.

**Comment by chosak** [comment]
@willbarton oh, I see. Somehow I didn't realize that `AbstractImage` was actually an _abstract_ model. Thanks for the explanation.

**Comment by willbarton** [comment]
Oh Codecov.

**Pull request merged**